{
  try {
    tester.doTestEquals(ConstructorThrows.class);
    fail("should have failed");
  }
 catch (  InvocationTargetException expected) {
  }
}
